CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated! Every little bit
+helps, and credit will always be given.
+
+## Cookiecutter Contribution vs Application Contribution
+
+Please note that this repository is only the cookiecutter and not the entire
+CDP tooling and infrastructure ecosystem. This repository ties all of our
+tooling together into a single easy to deploy and maintain repository.
+Contributions to this repository should largely be documentation,
+devops, bugfixes, or similar.
+
+If you experience a bug or incorrect documentation while using the cookiecutter
+please do send us a Pull Request! If you want to add or fix an auto-deployment
+bot, or add more GitHub Actions to the produced repository, all such contributions
+welcome and appreciated.
+
+Examples of these types of contributions include:
+
+-   adding more instance admin documentation to the generated repository
+-   updating the `cdp-backend` and `cdp-frontend` version pins
+-   upgrading or fixing and auto-deployment GitHub Action
+-   adding new GitHub Actions to the generated repository
+
+For contributions to the major pipelines and infrastructure that are used by all
+CDP deployments, please see:
+[cdp-backend](https://github.com/councildataproject/cdp-backend)
+
+For contributions to the web application which is used by all CDP deployments, please
```

Comment:
Maybe "web application components" instead?

Reply:
Ehhhh this one seems fine to me. I won't resolve this comment yet as I think others may have input on it but I currently like it how it is.

Reply:
Yeah +1 to Jackson since the frontend repo contains both the component library and web app code. 

So for the purposes of being concise I think "web application" gets the point across clearly.